browserprocess/register.go

```diff
@@ -0,0 +1,59 @@
+package browserprocess
+
+import (
+	"context"
+	"os"
+	"strconv"
+	"strings"
+	"sync"
+
+	"github.com/grafana/xk6-browser/log"
+)
+
+type processState struct {
+	pid int
+}
+
+var (
+	browserProcessRegister   = map[string]*processState{} //nolint:gochecknoglobals
```

Comment:
Can we simplify this map now that we don't need to store the `killed` status as `map[string]int` being: `pid+iID: pid` and get rid of the `processState` struct?

Reply:
Good point, although it's needed in the [next PR](https://github.com/grafana/xk6-browser/pull/762) that i've created 😅 Maybe i'll revert it here still to show that it's not needed for this particular change 👍